main/src/com/google/refine/model/recon/ReconciledDataExtensionJob.java

```diff
@@ -218,46 +203,17 @@ public ReconciledDataExtensionJob(DataExtensionConfig obj, String endpoint) {
         return map;
     }
 
-    /**
-     * @todo this should be refactored to be unified with the HTTP querying code
-     * from StandardReconConfig. We should ideally extract a library to query
-     * reconciliation services and expose it as such for others to reuse.
-     */
```

Comment:
I would leave this comment since this refactoring does not address this yet.

Reply:
My thought here was that the refactoring in this PR was a superset of the suggest recon-only client. Is that not true? What special requirements does reconciliation have? The code that I thought the comment was referring to is just a single line now. 

The Create Project streaming is a slightly different use, but it's already accommodated in the current code.

I can add a version of the comment back if it's still relevant, but perhaps we should update it to reflect the current state of affairs.

Reply:
I was thinking about a java library to query reconciliation services. You would be able to construct a reconciliation query with POJOs and send it to a service described by a URL (without having to worry about the JSON serialization of that reconciliation query, for instance). This is more high-level than a HTTP library: it would abstract away the specifics of how reconciliation clients and servers communicate over HTTP.

Such a library could also be the basis of a framework to build reconciliation services.

Reply:
That sounds like it has very little, if anything to do with OpenRefine, but I'll put the comment back with a clarification that it's talking about a 3rd party library.

Reply:
I've restored the comment, moved it so it's not on a single line method, and updated it with my limited understanding of what the task is. I've also rebased it to pick up the new CI code and will merge it when the tests pass.